crates/bevy_ui/src/render/mod.rs

```diff
@@ -553,41 +546,26 @@ pub fn extract_default_ui_camera_view(
 ) {
     live_entities.clear();
 
-    let scale = ui_scale.0.recip();
     for (entity, camera, ui_anti_alias, shadow_samples) in &query {
         // ignore inactive cameras
         if !camera.is_active {
             commands
                 .get_entity(entity)
                 .expect("Camera entity wasn't synced.")
-                .remove::<(DefaultCameraView, ExtractedAA, UiBoxShadowSamples)>();
+                .remove::<(DefaultCameraView, UiAntiAlias, UiBoxShadowSamples)>();
             continue;
         }
 
-        if let (
-            Some(logical_size),
-            Some(URect {
-                min: physical_origin,
-                ..
-            }),
-            Some(physical_size),
-            Some(scale_factor),
-        ) = (
-            camera.logical_viewport_size(),
-            camera.physical_viewport_rect(),
-            camera.physical_viewport_size(),
-            camera.target_scaling_factor(),
-        ) {
+        if let (Some(physical_viewport_rect),) = (camera.physical_viewport_rect(),) {
```

Comment:
whats the point of the tuple on either side of this let?
```suggestion
        if let Some(physical_viewport_rect) = camera.physical_viewport_rect() {
```